pkg/client/cli/cmd/list_namespaces.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli/ann"
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli/connect"
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli/daemon"
+	"github.com/telepresenceio/telepresence/v2/pkg/client/cli/output"
+)
+
+type listNamespacesCommand struct {
+	rq *daemon.Request
+}
+
+func listNamespaces() *cobra.Command {
+	lnc := &listNamespacesCommand{}
+
+	cmd := &cobra.Command{
+		Use:   "list-namespaces",
+		Args:  cobra.NoArgs,
+		Short: "Show all namespaces",
+		RunE:  lnc.run,
+		Annotations: map[string]string{
+			ann.UpdateCheckFormat: ann.Tel2,
+		},
+	}
+	lnc.rq = daemon.InitRequest(cmd)
+	return cmd
+}
+
+func (lnc *listNamespacesCommand) run(cmd *cobra.Command, _ []string) error {
+	if err := connect.InitCommand(cmd); err != nil {
+		return err
+	}
```

Comment:
No need for `InitCommand` here (it's not called when doing auto-completion either).